couchdb/server/__init__.py

```diff
@@ -253,6 +253,27 @@ def log(self, message):
             res = ['log', message]
         self.respond(res)
 
+    def compile(self, funsrc, ddoc=None, context=None, **options):
+        """Compiles function with special server context.
+
+        :param funsrc: Function source code.
+        :type funsrc: str
+
+        :param ddoc: Design document object.
+        :type ddoc: dict
+
+        :param context: Custom context for compiled function.
+        :type context: dict
+
+        :param options: Compiler config options.
+        """
+        if context is None:
+            context = {}
+
+        context.setdefault('log', self.log)
```

Comment:
That's not a point to care since you can access to whole stdlib, file system and everything else. Sandboxing Python is not a trivial task and every user should keep in mind, that Python query server is unsafe for public usage (as like as Erlang one and else non-sandboxed).